intellij-lsp/src/com/github/gtache/lsp/requests/HoverHandler.scala

```diff
@@ -24,19 +24,22 @@ object HoverHandler {
       if (hoverContents.isLeft) {
         val contents = hoverContents.getLeft.asScala
         if (contents == null || contents.isEmpty) "" else {
-          contents.map(c => {
+          val stuff = contents.map(c => {
```

Comment:
Something like `parsedContents` would be better

Reply:
Renamed as `parsedContents`